ApiQuotaHelper.php

```diff
@@ -0,0 +1,41 @@
+<?php
+/**
+ * Matomo - free/libre analytics platform
+ *
+ * @link https://matomo.org
+ * @license http://www.gnu.org/licenses/gpl-3.0.html GPL v3 or later
+ */
+namespace Piwik\Plugins\GoogleAnalyticsImporter;
+
+/**
+ * Utility class with methods to manage Google Analytics Importer INI configuration.
+ */
+class ApiQuotaHelper
+{
+    /**
+     * Calculate the daily maximum api quota available for the instance
+     * @return int
+     */
+    public function getMaxDailyApiQuota($setTrial = false)
```

Comment:
It doesn't look like `$setTrial` is used for anything. Is there a reason it's there?